### PR TITLE
Doshinirav new option --message-file /path/filename

### PR DESCRIPTION
**Pull request by doshinirav**
This option allows taking the issue comments from an existing file.

### CODE REVIEW

File: jiracli/__init__.py [hunk 1]
```diff
@@ -19,6 +19,7 @@
 from __future__ import print_function
 
 import os
+import os.path
```
Comment (toabctl) [inline jiracli/__init__.py]: not needed. you can just use os.path in the code

File: jiracli/__init__.py [hunk 2]
```diff
@@ -78,7 +87,7 @@ def config_credentials_get():
 def sprint(jira_obj, project):
     issues = jira_obj.search_issues(
         'project = "%s" AND sprint IN openSprints()' % project)
-    height, width = [
+    width, height = [
```
Comment (toabctl) [inline jiracli/__init__.py]: ? @dam5h fixed that. Why do you want to revert it?

File: jiracli/__init__.py [hunk 3]
```diff
@@ -54,6 +55,14 @@ def setup_logging(logger, debug):
     logger.addHandler(sh)
 
 
+def file_get_text(fname):
+    # get text from an a provided filename
+    tf = open(fname, 'r')
```
Comment (toabctl) [inline jiracli/__init__.py]: please use a context manager to open the file: ```with open(fname, 'r') as f: ...```

**Comment by toabctl** [comment]
In general, your commits are wrong. You have 4 commits without describing in each commit what the commit is good for and *why* you are changing something. Can you please fix that?

**Comment by toabctl** [comment]
@doshinirav ping ... still interessted in that?

**Comment by doshinirav** [comment]
Yes I am interested, will make the changes later today

**Comment by toabctl** [comment]
@doshinirav friendly ping ...

**Comment by toabctl** [comment]
Feel free to reopen the pull request when you have time to work on this.

**Pull request closed without merging**